features/admin.extensions.v1/configs/common.tsx

```diff
@@ -117,6 +118,20 @@ export const commonConfig: CommonConfig = {
                 path: AppConstants.getPaths().get("OUTBOUND_PROVISIONING_SETTINGS"),
                 protected: true,
                 showOnSidePanel: false
+            },
+            {
+                category: "extensions:manage.sidePanel.categories.userManagement",
```

Comment:
Any reason why this is added in extensions side, can't we define this in routes.tsx itself?